lab-dustin/lib/readfile.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const { readFile } = require('fs');
+
+exports.readData = function(file, callback) {
+  readFile(`${__dirname}/../assets/${file}`, function(err, buff){
+    if (err) { callback(err); }
+    console.log(file);
+    console.log(buff.toString('hex').slice(0,8));
+    if(callback) { callback(null,
```

Comment:
Formatting here is a bit weird - unless you're going to open and close `{ }` on the same line, put a newline after `{`.